python/desc/twinkles/sqlite_tools.py

```diff
@@ -29,3 +31,30 @@ def create(self, columns, table, condition=None):
             query += ' ' + condition
         data = [row for row in self.conn.execute(query)]
         return pd.DataFrame(data=data, columns=columns)
+
```

Comment:
Not particularly.  I'm just not very familiar with the available pandas functions.  I guess in this case, `pd.read_sql_query` would be the one to use assuming we really didn't want to ingest the entire table?